packages/frontend/web/components/BigNumber/0.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="34" height="40">
```

Comment:
All of these probably should live with the pasteup icons that I've moved to the frontend package under `static/` but that PR isn't merged yet so I'll add that once this is merged (or when they both are).